Dockerfile

```diff
@@ -19,28 +24,21 @@ RUN mkdir /projects ${HOME} && \
       chmod -R g+rwX ${f}; \
     done
 
-# odo and oc versions have to match the ones defined in https://github.com/redhat-developer/vscode-openshift-tools/blob/master/src/tools.json
-ENV GLIBC_VERSION=2.30-r0 \
-    ODO_VERSION=v1.0.1 \
-    OC_VERSION=v3.11.0 \
-    OC_TAG=0cbc58b \
-    KUBECTL_VERSION=v1.16.2
-
 # plugin executes the commands relying on Bash
 RUN apk add --update --no-cache bash && \
     # install glibc compatibility layer package for Alpine Linux
     # see https://github.com/openshift/origin/issues/18942 for the details
     wget -O glibc-${GLIBC_VERSION}.apk https://github.com/andyshinn/alpine-pkg-glibc/releases/download/${GLIBC_VERSION}/glibc-${GLIBC_VERSION}.apk && \
     apk --update --allow-untrusted add glibc-${GLIBC_VERSION}.apk && \
```

Comment:
Shouldn't we also use `--no-cache` here?

Reply:
Of course the GitHub UI has bested me once again. This comment was intended for the `apk --update --allow-untrusted add glibc-${GLIBC_VERSION}.apk` line.